app/front/scripts/services/validation.js

```diff
@@ -1,47 +1,22 @@
 'use strict';
 
 var _ = require('lodash');
-var utils = require('../../../services/utils');
+var goodtables = require('goodtables');
 
-var goodTablesUrl = '//goodtables.okfnlabs.org/api/run';
+var goodtablesOptions = {
+  apiUrl: 'https://goodtables.io/api',
+  apiToken: 'D0123458B8E36326C60253FE4A7FF6662CAB0C48',
+  apiSourceId: '9b6b6391-5404-4e7f-bdb8-271c2cb42fbb'
```

Comment:
I guess this shouldn't be here?

Reply:
It's publicly available `goodtables.io` credentials to use with `goodtables-js` in a "guest" mode.

Reply:
@roll Cool, let's go with it for now. I opened an issue on https://github.com/frictionlessdata/goodtables.io/issues/279 for us to discuss this further.